recipes/dynamo-release/meta.yaml

```diff
@@ -0,0 +1,74 @@
+{% set name = "dynamo-release" %}
+{% set version = "1.3.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/aristoteleo/{{ name }}/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: 4b9e0b9721802b0b7baa05a0651518958d512660703ca0d62a5140a7689f3d22
+
+build:
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+  noarch: python
+
+requirements:
+  build:
+    - python >=3.7
+  host:
+    - python >=3.7
+    - numpy 1.23.*
+    - markdown 3.3.*
+    - setuptools
+    - pip
+  run:
+    - python >=3.7
+    - setuptools
+    - numpy 1.23.*
+    - pandas >=1.3.5
+    - scipy >=1.4.1
+    - scikit-learn >=0.19.1
+    - anndata >=0.8.0
+    - loompy >=3.0.5
+    - matplotlib-base >=3.5.3
+    - more-itertools
+    - get_version >=3.5.4
+    - numdifftools >=0.9.39
+    - umap-learn >=0.5.1
+    - PATSY
+    - statsmodels
+    - numba >=0.54.0
+    - seaborn >=0.9.0
+    - colorcet >=2.0.1
+    - tqdm
+    - igraph >=0.7.1
+    - louvain ==0.8.0
+    - pynndescent >=0.5.2
+    - pre-commit
+    - networkx
+    - typing-extensions
+    - openpyxl
+    - session-info >=1.0.0
+
+test:
+  imports:
+    - dynamo
+  requires:
+    - pip
+#  commands:
+#    - pip check # fails probably because package python-igraph has a different name in pip
+
+about:
+  home: https://github.com/aristoteleo/dynamo-release
+  summary: 'Inclusive model of expression dynamics with metabolic labeling based scRNA-seq'
+  description: |
+    Inclusive model of expression dynamics with metabolic labeling based scRNA-seq in silico perturbation predictions.
+  license: MIT
+  license_family: MIT
+  license_file: LICENSE
+
+extra:
+  recipe-maintainers:
+    - Xiaojieqiu
```

Comment:
@Ukyeon and @Sichao25 This means that you won't be a maintainer, is that correct?